cogboard-app/src/test/kotlin/com/cognifide/cogboard/TestHelper.kt

```diff
@@ -0,0 +1,12 @@
+package com.cognifide.cogboard
+
+import io.vertx.core.json.JsonObject
+
+class TestHelper {
+    companion object {
+        fun readConfigFromResource(pathToJsonResource: String) : JsonObject {
+            val fileContent = TestHelper::class.java.getResource(pathToJsonResource).readText()
+            return JsonObject(fileContent.trimIndent())
```

Comment:
Is `.trimIndent()` strictly necessary?

Reply:
don't think so - this code was just moved not changed.